src/mainwindow.ui

```diff
@@ -116,19 +116,28 @@
     <property name="title">
      <string>&amp;View</string>
     </property>
+    <widget class="QMenu" name="menuIcon_Size">
+     <property name="title">
+      <string>&amp;Icon Size</string>
+     </property>
+     <addaction name="action16px"/>
+     <addaction name="action24px"/>
+     <addaction name="action32px"/>
+     <addaction name="action48px"/>
+    </widget>
     <addaction name="separator"/>
     <addaction name="actionToolbar"/>
     <addaction name="actionStatusbar"/>
     <addaction name="actionDirTree"/>
     <addaction name="separator"/>
+    <addaction name="menuIcon_Size"/>
     <addaction name="actionFlatListMode"/>
     <addaction name="actionDirTreeMode"/>
     <addaction name="separator"/>
     <addaction name="actionExpand"/>
     <addaction name="actionCollapse"/>
     <addaction name="separator"/>
     <addaction name="actionStop"/>
-    <addaction name="actionReload"/>
```

Comment:
Why did you remove this?

Reply:
i didn't, its qtcreator doing some weird stuff, will fix.